.changeset/cyan-drinks-enjoy.md

```diff
@@ -0,0 +1,7 @@
+---
+"@comet/admin": major
+---
+
+Use `permanentDrawer` and `temporaryDrawer` slots instead of `drawer` slot
+
+Delete `permanentDrawerProps` and `temporaryDrawerProps`, use `slotProps` instead
```

Comment:
Is this change related?

Reply:
I'm not sure, if I understand the question correctly, but yes: the new slots relate to the deleted props.

Reply:
Yes, this is better!